app/bounty_requests/templates/bounty_request_form.html

```diff
@@ -38,46 +38,36 @@ <h3>{% trans "Request a Bounty" %} <span id="alpha">{% trans "Alpha" %}</span></
                   Suggest it be given a bounty!
                   {% endblocktrans %}
                 </p>
-                <form id='bounty_request_form'>
-                  <div class="mb-3">
-                    <label for="requested_by" class="form__label">{% trans "Your Github Profile" %}
-                      {% if not user.is_authenticated %}
-                      (<a href="{% url 'social:begin' 'github' %}?next={{ request.get_full_path }}">{% trans "Login" %}</a>)
-                      {% endif %}
-                    </label>
-                    <input name="requested_by" placeholder="{% trans "Please Login First" %}" id="requested_by" class="form__input" type="text" value="{% if github_handle %}{{ github_handle }}{% endif %}" required disabled />
-                  </div>
-                  <div class="mb-3">
-                    <label for="github_url" class="form__label">{% trans "The Github Issue Link" %}</label>
-                    <input name="github_url" placeholder="https://github.com/gitcoinco/web/issues/2036" id="github_url" class="form__input" type="text" required />
-                  </div>
-                  <div class="mb-3">
-                    <label for="eth_address" class="form__label">{% trans "Your ETH Address (Optional)" %}</label>
-                    <input name="eth_address" placeholder="0x0" id="eth_address" class="form__input" type="text" />
-                  </div>
-                  <div class="mb-3">
-                    <label for="amount" class="form__label">{% trans "Proposed Funding Amount (USD)" %}</label>
-                    <input name="amount" placeholder="1" id="amount" class="form__input" type="number" min="1" required />
-                  </div>
-                  <div class="mb-3">
-                    <label for="comment" class="form__label"> {% trans "Comment" %} (<span id="charcount">500</span> {% trans "characters left" %}):</label>
-                    <textarea name="comment" id="comment" class="form__input" rows="4" cols="50" required placeholder="{% trans "Anything you want to tell us." %}"></textarea>
-                  </div>
+                <div class="mb-3">
+                  <label for="requested_by" class="form__label">{% trans 'Your Github Profile' %}
+                    {% if not user.is_authenticated %}
+                    (<a href="{% url 'social:begin' 'github' %}?next={{ request.get_full_path }}">{% trans 'Login' %}</a>)
+                    {% endif %}
+                  </label>
+                  <input name="requested_by" placeholder="{% trans "Please Login First" %}" id="requested_by" class="form__input" type="text" value="{% if github_handle %}{{ github_handle }}{% endif %}" required disabled />
+                </div>
+                <form id="bounty_request_form">
+                  {% for field in form %}
+                    <div class="mb-3">
+                      <label for="{{ field.name }}" class="form__label">{{ field.label }}</label>
```

Comment:
The `for` attribute binds it to the id of the element and not the name. So for this to work, I think you need to set the id for the elements same as the name in the form widget.

Reply:
@SaptakS Whoa. Yes you are right, seems like I'm getting too old 😆 .

Reply:
😅